spec/app/domain/authentication/authn-oidc/oidc_spec.rb

```diff
@@ -224,5 +271,35 @@
                               )
       end
     end
+
+    context "that receives authenticate request" do
```

Comment:
Could you jsut add `"and fails on oidc details retrieval"` to the end of this description to be consistent with the one above?